addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -180,7 +180,7 @@ const CountdownWidget = publicWidget.Widget.extend({
         }
         this._updateTimeDiff();
 
-        const hideCountdown = this.isFinished && !this.editableMode && this.$el.hasClass('hide-countdown');
```

Comment:
Unfortunately, don't rename that class. Otherwise old countdown used by customers will not work anymore.